docs/resources/feature.md

```diff
@@ -0,0 +1,91 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "flagsmith_feature Resource - terraform-provider-flagsmith"
+subcategory: ""
+description: |-
+  Flagsmith Feature/ Remote config
+---
+
+# flagsmith_feature (Resource)
+
+Flagsmith Feature/ Remote config
+
+## Example Usage
+
+```terraform
+resource "flagsmith_feature" "new_mv_feature" {
+  feature_name = "new_mv_feature"
+  project_uuid = "10421b1f-5f29-4da9-abe2-30f88c07c9e8"
+  description  = "This is a new multivariate feature"
+  type         = "MULTIVARIATE"
+  multivariate_options = [
```

Comment:
Thinking about it - we deprecated creating multivariate_options for most users, right? I appreciate that we still allow project admins to do it but I'm not sure we should be including it here, right?